test/arrays/bradc/formalActualArrayArgMatch.chpl

```diff
@@ -0,0 +1,45 @@
+use BlockDist;
```

Comment:
Yeah, I guess.

The main advantage to suppressing is that we're checking that we do actually turn off these potentially expensive runtime checks with --fast. I go back and forth on whether I think suppressions or skipping is the right solution, so I don't object to skipping if it's easier. Your skipif text does need to be updated though, since it talks about why we suppress rather than skip